### PR TITLE
[bitnami/influxdb] ingress/tls hostname template support

### PR DESCRIPTION
**Pull request by chlumper**
Signed-off-by: Christian Lumper <ch.lumper@gmail.com>

**Description of the change**

Add support for template directives at hostname of ingress/tls configuration.

**Benefits**

Allows to use values for generation of hostname which is especially handy if chart is used within umbrella chart.
e.g.
```yaml
ingress:
  enabled: true
  tls: true
  hostname: '{{ .Values.global.domain }}'
```

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/influxdb/Chart.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ name: influxdb
 sources:
   - https://github.com/bitnami/bitnami-docker-influxdb
   - https://www.influxdata.com/products/influxdb-overview/
-version: 4.0.0
+version: 4.1.0
```
Comment (juan131) [inline bitnami/influxdb/Chart.yaml:27]: Could you please bump the patch version instead?
```suggestion
4.0.1
```

**Review by juan131 — COMMENTED** [review]
Thanks so much for this great contribution!

File: bitnami/influxdb/Chart.yaml (hunk 1 above)
Comment (chlumper) [reply bitnami/influxdb/Chart.yaml]: Adapted to only bump revision (4.0.1).

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**